Recrutify/Recrutify.DataAccess/Repositories/UserRepository.cs

```diff
@@ -35,5 +37,18 @@ public Task<User> GetByEmailAsync(string email)
             var filter = _filterBuilder.Eq(u => u.Email, email);
             return GetCollection().Find(filter).FirstOrDefaultAsync();
         }
+
+        public async Task<IEnumerable<User>> GetByRoles(List<Role> roles)
+        {
```

Comment:
can write var filterBuilder = Builders<BsonDocument>
                            .Filter;
and use in other places

Reply:
+